src/components/Comparisons/RelatedItem.jsx

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
```

Comment:
Same note as OutfitItem, might be safer to at least define this before exporting.

Reply:
defined it!